composer.json

```diff
@@ -7,13 +7,19 @@
     "autoload-dev": {
         "psr-4": {
             "App\\Spec\\": "spec/"
-        }
+        },
+        "files": [
+          "src/procedural.php"
+        ]
     },
     "config": {
         "bin-dir": "bin"
     },
     "minimum-stability": "dev",
     "prefer-stable": true,
+    "require": {
+      "php": "^5.5|^5.6"
```

Comment:
Resolved. Will skip CI on remaining feedback items until we can get the follow-ups from https://github.com/sitrunlab/learn-kahlan/pull/9#issuecomment-302755346 cleared.